R/reg_intervals.R

```diff
@@ -0,0 +1,105 @@
+#' A convenience function for confidence intervals with linear-ish parametric models
+#'
+#' @param formula An R model formula with one outcome and at least one predictor.
+#' @param data A data frame.
+#' @param model_fn The model to fit. Allowable values are "lm", "glm",
+#'  "survreg", and "coxph". The latter two require that the `survival` package
+#'  be installed.
+#' @param type The type of bootstrap confidence interval. Values of "student-t" and
+#' "percentile" are allowed.
+#' @param times A single integer for the number of bootstrap samples. If left
+#' NULL, 1,001 are used for t-intervals and 2,001 for percentile intervals.
+#' @param alpha Level of significance.
+#' @param filter A logical statement to filter the model terms (or NULL).
```

Comment:
```suggestion
#' @param filter A logical expression used to remove rows from the final result, or `NULL` to keep all rows.
```

You might want to tweak it a little more, but generally I wanted to get at the fact that this removes _rows_ from the final result using a `filter()`